platformio.ini

```diff
@@ -10,23 +10,25 @@
 
 [common]
 board_f_cpu = 160000000L
-lib_deps_builtin = 
+lib_deps_builtin =
   SPI
-lib_deps_external = 
+lib_deps_external =
   RF24
   WiFiManager
   ArduinoJson
-build_flags = !echo -D MILIGHT_HUB_VERSION=\\\"$(git describe --always)\\\"
+build_flags = !echo -D MILIGHT_HUB_VERSION="\"$(git describe --always)\""
+; build_flags = -D MILIGHT_HUB_VERSION=\"1.1.0\" -D MILIGHT_UDP_DEBUG
 # -D MILIGHT_UDP_DEBUG
+upload_speed = 9600
```

Comment:
Was flashing not working with the default baudrate? What board are you using? Looks like d1_mini? I've flashed a few different d1_minis with the default baudrate without any trouble.